docs/source/quickstart.mdx

```diff
@@ -73,107 +90,41 @@ The next step is to tokenize the text in order to build sequences of integers th
-'label': tensor([1, 0, 1, 0, 1, 1, 0, 1]),
-'token_type_ids': tensor([[0, 0, 0,  ..., 0, 0, 0],
-                     [0, 0, 0,  ..., 0, 0, 0],
-                     [0, 0, 0,  ..., 0, 0, 0],
-                     ...,
-                     [0, 0, 0,  ..., 0, 0, 0],
-                     [0, 0, 0,  ..., 0, 0, 0],
-                     [0, 0, 0,  ..., 0, 0, 0]])}
 ```
 </pt>
 <tf>
-1. Rename the `label` column to `labels`, the expected input name in [TFBertForSequenceClassification](https://huggingface.co/transformers/model_doc/bert#tfbertforsequenceclassification):
-
-```py
->>> dataset = dataset.map(lambda examples: {'labels': examples['label']}, batched=True)
-```
-
-2. Retrieve the actual tensors from the Dataset object instead of using the current Python objects.
-3. Filter the dataset to only return the model inputs: `input_ids`, `token_type_ids`, and `attention_mask`.
-
-[`Dataset.set_format`] completes the last two steps on-the-fly. After you set the format, wrap the dataset in `tf.data.Dataset`:
-
+Wrap the dataset in [`tf.data.Dataset`](https://www.tensorflow.org/api_docs/python/tf/data/Dataset):
 
 ```py
 >>> import tensorflow as tf
->>> dataset.set_format(type='tensorflow', columns=['input_ids', 'token_type_ids', 'attention_mask', 'labels'])
->>> features = {x: dataset[x].to_tensor(default_value=0, shape=[None, tokenizer.model_max_length]) for x in ['input_ids', 'token_type_ids', 'attention_mask']}
->>> tfdataset = tf.data.Dataset.from_tensor_slices((features, dataset["labels"])).batch(32)
->>> next(iter(tfdataset))
-({'input_ids': <tf.Tensor: shape=(32, 512), dtype=int32, numpy=
-array([[  101,  7277,  2180, ...,     0,     0,     0],
-   [  101, 10684,  2599, ...,     0,     0,     0],
-   [  101,  1220,  1125, ...,     0,     0,     0],
-   ...,
-   [  101,  1109,  2026, ...,     0,     0,     0],
-   [  101, 22263,  1107, ...,     0,     0,     0],
-   [  101,   142,  1813, ...,     0,     0,     0]], dtype=int32)>, 'token_type_ids': <tf.Tensor: shape=(32, 512), dtype=int32, numpy=
-array([[0, 0, 0, ..., 0, 0, 0],
-   [0, 0, 0, ..., 0, 0, 0],
-   [0, 0, 0, ..., 0, 0, 0],
-   ...,
-   [0, 0, 0, ..., 0, 0, 0],
-   [0, 0, 0, ..., 0, 0, 0],
-   [0, 0, 0, ..., 0, 0, 0]], dtype=int32)>, 'attention_mask': <tf.Tensor: shape=(32, 512), dtype=int32, numpy=
-array([[1, 1, 1, ..., 0, 0, 0],
-   [1, 1, 1, ..., 0, 0, 0],
-   [1, 1, 1, ..., 0, 0, 0],
-   ...,
-   [1, 1, 1, ..., 0, 0, 0],
-   [1, 1, 1, ..., 0, 0, 0],
-   [1, 1, 1, ..., 0, 0, 0]], dtype=int32)>}, <tf.Tensor: shape=(32,), dtype=int64, numpy=
-array([1, 0, 1, 0, 1, 1, 0, 1, 0, 0, 0, 0, 1, 1, 0, 0, 0, 1, 0, 1, 1, 1,
-   0, 1, 1, 1, 0, 0, 1, 1, 1, 0])>)
+
+>>> dataset.set_format(type="tensorflow", columns=["input_ids", "token_type_ids", "attention_mask", "labels"])
+>>> features = {x: dataset[x] for x in ["input_ids", "token_type_ids", "attention_mask"]}
+>>> tf_dataset = tf.data.Dataset.from_tensor_slices((features, dataset["labels"])).batch(32)
```

Comment:
We now have a memory efficient `to_tf_dataset`, maybe we may mention it in the quickstart instead.

`from_tensor_slices` brings everything in memory